playwright.config.js

```diff
@@ -104,10 +103,10 @@ export default defineConfig({
   outputDir: 'src/e2e/test-results/',
 
   /* Run your local dev server before starting the tests */
-  webServer: {
-    command: 'npm run build; npm start',
-    port: 6060,
-    // Building the app can take some time:
-    timeout: 1_800_000,
-  }
+  // webServer: {
+  //   command: 'npm run build; npm start',
+  //   port: 6060,
+  //   // Building the app can take some time:
+  //   timeout: 1_800_000,
+  // }
```

Comment:
Q: Do we not need this anymore, or is this just a temporary commenting-out?

Reply:
ah accident